lib/utils/analytics/getSDKVersions.js

```diff
@@ -2,17 +2,28 @@ const fs = require('fs');
 const path = require('path');
 const sdkCode = 'M'; // Constant per SDK
 
+function readSdkSemver() {
+  const pkgJsonPath = path.join(__dirname, '../../../package.json');
+  try {
+    const pkgJSONFile = fs.readFileSync(pkgJsonPath, 'utf-8');
+    return JSON.parse(pkgJSONFile).version
+  } catch (e) {
+    if (e.code === 'ENOENT') {
+      return '0.0.0'
+    }
+    return 'n/a';
```

Comment:
Is there a test for `n/a`? I would change it to something we can decode or would not cause additional errors.

Reply:
Yes, there is - the resulting token will be `_a=E` because it assumes numerical values